frontend/i18next.scanner/en/translation.json

```diff
@@ -262,16 +263,16 @@
   "Error while changing username": "Error while changing username",
   "Username and name has been changed": "Username and name has been changed",
   "Username has been changed": "Username has been changed",
-  "Personal data has been changed": "Personal data has been changed",
-  "Invalid email or username": "Invalid email or username",
-  "Email address or username": "Email address or username",
+  "Personal data has been changed": "Personal data changed",
+  "Invalid email or username": "Invalid e-mail address or username",
+  "Email address or username": "E-mail address or username",
   "Invalid credentials": "Invalid credentials",
-  "Please type your password in order to change your email and/or username. (For security reasons)": "Please type your password in order to change your email and/or username. (For security reasons)",
+  "Please type your password in order to change your email and/or username. (For security reasons)": "Please type your password to change your e-mail address and/or username. (For security reasons.)",
   "Please confirm your password:": "Please confirm your password:",
-  "Unknown email or username": "Unknown email or username",
-  "This username is not linked to any email, please contact an administrator to reset your password": "This username is not linked to any email, please contact an administrator to reset your password",
-  "{{filename}} is bigger than 1MB, this may take a while to upload, do you want to continue?": "{{filename}} is bigger than 1MB, this may take a while to upload, do you want to continue?",
-  "Tracim has a connection problem, please wait or restart your browser if the problem persists": "Tracim has a connection problem, please wait or restart your browser if the problem persists",
+  "Unknown email or username": "Unknown e-mail address or username",
+  "This username is not linked to any email, please contact an administrator to reset your password": "This username is not linked to any e-mail address. Please contact an administrator to reset your password.",
+  "{{filename}} is bigger than 1MB, this may take a while to upload, do you want to continue?": "Du you want to continue uploading {{filename}} even though it is bigger than 1 MB?",
```

Comment:
```suggestion
  "{{filename}} is bigger than 1MB, this may take a while to upload, do you want to continue?": "Do you want to continue uploading {{filename}} even though it is bigger than 1 MB?",
```

Reply:
Changed.